.claude/skills/deps-updater/scripts/tests/test-discover-deps.py

```
@@ -0,0 +1,106 @@
+        [sys.executable, str(SCRIPT), "--help"],
+        capture_output=True, text=True,
+    )
+    assert result.returncode == 0
+    assert "--workspace" in result.stdout
+
+
+def test_empty_project():
+    with tempfile.TemporaryDirectory() as tmp:
+        tmp = Path(tmp)
+        (tmp / "package.json").write_text('{"name":"test","private":true}')
+        result = subprocess.run(
```

Ruff (0.15.17)

[error] `subprocess` call: check for execution of untrusted input

(S603)